packages/ui/src/components/Channel/Scroller.tsx

```diff
@@ -505,6 +505,33 @@ const BaseScrollerItem = ({
       <ChannelDivider unreadCount={0} post={post} index={index} />
     ) : null;
 
+  if (post.isDeleted) {
```

Comment:
could we make the press handlers no ops instead of duplicating the whole `View`? If not for some reason, maybe a dynamic wrapper component that we set above